Sources/Rendering/SceneGraph/ViewNode/index.js

```diff
@@ -113,14 +113,27 @@ function vtkViewNode(publicAPI, model) {
 
   // add missing nodes/children for the passed in renderables. This should
   // be called only in between prepareNodes and removeUnusedNodes
-  publicAPI.addMissingNodes = (dataObjs) => {
+  publicAPI.addMissingNodes = (dataObjs, enforceOrder = false) => {
     if (!dataObjs || !dataObjs.length) {
       return;
     }
 
     for (let index = 0; index < dataObjs.length; ++index) {
       const dobj = dataObjs[index];
-      publicAPI.addMissingNode(dobj);
+      const node = publicAPI.addMissingNode(dobj);
```

Comment:
shouldn't you propagate `enforceOrder`?

Reply:
IIUC, `addMissingNode` will create a node for the dataobject and the following code will ensure it gets spliced in at the right index. So, not sure why `addMissingNode` needs to know about `enforceOrder`.

Reply:
My bad, I thought it was a recursive function (hence propagation), but it is not addMissingNodes != addMissingNode